internal/codegen/golang/driver.go

```diff
@@ -3,14 +3,16 @@ package golang
 type SQLDriver int
 
 const (
-	SQLPackagePGXV4    string = "pgx/v4"
-	SQLPackagePGXV5    string = "pgx/v5"
-	SQLPackageStandard string = "database/sql"
+	SQLPackagePGXV4            string = "pgx/v4"
+	SQLPackagePGXV5            string = "pgx/v5"
```

Comment:
I'm starting to think we should have called this `jackc/pgx/v5` or `github.com/jackc/pgx/v5`. Happy to keep `go-sql-driver/mysql` for now.

Reply:
Easy enough to start accepting the new values as well as the old values at some point, right?